### PR TITLE
bug fix for vet tec profile page mobile view styling

### PR DESCRIPTION
**Pull request by hughes-dustin**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19410

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]
Approved by QA

**Comment by hughes-dustin** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

**Comment by hughes-dustin** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/gi/components/vet-tec/VetTecCalculator.jsx [hunk 1]
```diff
@@ -165,10 +165,12 @@ export class VetTecCalculator extends React.Component {
...
     return (
       <div className="vads-l-row calculate-your-benefits">
-        <div className="medium-5 columns">{this.renderCalculatorForm()}</div>
-        <div className="medium-1 columns">&nbsp;</div>
-        <div className="medium-6 columns vads-u-margin-left--2 vads-u-margin-right--neg2">
-          <div className="your-estimated-benefits">
+        <div className="usa-width-five-twelfths medium-5 columns">
+          {this.renderCalculatorForm()}
+        </div>
+        <div className="usa-width-one-twelfth columns">&nbsp;</div>
```
Comment (U-DON) [inline src/applications/gi/components/vet-tec/VetTecCalculator.jsx]: This is missing the `medium-1` class now, but the other two columns kept their `medium-*` classes. Maybe those classes should be removed from all of the columns, or `medium-1` should be added back to this one?

Comment (hughes-dustin) [reply src/applications/gi/components/vet-tec/VetTecCalculator.jsx]: Okay. I added it back in.

**Review by U-DON — APPROVED** [review]

**Pull request merged**